app/views/directives/mobile-service-clients.html

```diff
@@ -0,0 +1,12 @@
+<div class="row mobile-service-clients-integrations">
+  <div class="col-sm-6">
+    <div ng-if="$ctrl.integratedClients | size" class="client-title component-label section-label">Mobile Clients</div>
+    <div ng-repeat="mobileClient in $ctrl.integratedClients track by $index" class="col-sm-12 col-md-6 mobile-client-integration">
+      <span aria-hidden="true" class="fa-2x {{mobileClient.metadata.annotations.icon}} client-icon"></span>
```

Comment:
Curious if this html/css should be made generic, rather than for `mobile-client`/`client-icon`.  Is it possible other components could reuse this pattern?  

@cshinn